### ISSUE TITLE
At Cannot find module 'tree-sitter-rust' in 

### ISSUE THREAD

**Issue by YoshioKimura** [atom/atom#19184]
**Atom**: 1.38.0-dev-1907d2da4 x64
**Electron**: 2.0.18
**OS**: Mac OS X 10.13.6
**Thrown From**: [language-rust-bundled](https://github.com/atom/atom) package 0.1.0

### Stack Trace

Failed to load a language-rust-bundled package grammar

```
At Cannot find module 'tree-sitter-rust' in /Applications/Atom Dev.app/Contents/Resources/app.asar/node_modules/language-rust-bundled/grammars/tree-sitter-rust.json

Error: Cannot find module 'tree-sitter-rust'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (/Applications/Atom Dev.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/Applications/Atom Dev.app/Contents/Resources/app/static/<embedded>:11:155153)
    at new TreeSitterGrammar (/Applications/Atom Dev.app/Contents/Resources/app/static/<embedded>:11:3343625)
    at GrammarRegistry.createGrammar (/Applications/Atom Dev.app/Contents/Resources/app/static/<embedded>:11:360834)
    at l.readFile (/Applications/Atom Dev.app/Contents/Resources/app/static/<embedded>:11:360675)
    at u (/Applications/Atom Dev.app/Contents/Resources/app/static/<embedded>:11:580425)
    at /Applications/Atom Dev.app/Contents/Resources/app/static/<embedded>:11:581678
    at ELECTRON_ASAR.js:530:9
    at FSReqWrap.wrapper [as oncomplete] (fs.js:658:17)
  at /app.asar/node_modules/language-rust-bundled/grammars/tree-sitter-rust.json:1:1
```

### Non-Core Packages

```
advanced-open-file 0.16.6 
atom-beautify 0.30.5 
atom-html-preview 0.1.22 
autocomplete-paths 1.0.2 
color-picker 2.2.2 
dotinstall-pane 0.1.8 
emmet 2.4.3 
export-html 3.1.0 
git-plus 5.23.2 
highlight-selected 0.11.2 
php-cs-fixer 4.1.0 
teletype 0.2.3 
```

**Comment by jasonrudolph** [comment]
@YoshioKimura: Thanks for reporting this issue.

I'm seeing this issue as well:

**Atom**: 1.38.0-nightly7 x64
**Electron**: 2.0.18
**OS**: Mac OS X 10.14.4
**Thrown From**: [language-rust-bundled](https://github.com/atom/atom) package 0.1.0

### Stack Trace

Failed to load a language-rust-bundled package grammar

```
At Cannot find module 'tree-sitter-rust' in /Applications/Atom Nightly.app/Contents/Resources/app.asar/node_modules/language-rust-bundled/grammars/tree-sitter-rust.json

Error: Cannot find module 'tree-sitter-rust'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (/Applications/Atom Nightly.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/Applications/Atom Nightly.app/Contents/Resources/app/static/<embedded>:11:155153)
    at new TreeSitterGrammar (/Applications/Atom Nightly.app/Contents/Resources/app/static/<embedded>:11:3343080)
    at GrammarRegistry.createGrammar (/Applications/Atom Nightly.app/Contents/Resources/app/static/<embedded>:11:360834)
    at l.readFile (/Applications/Atom Nightly.app/Contents/Resources/app/static/<embedded>:11:360675)
    at u (/Applications/Atom Nightly.app/Contents/Resources/app/static/<embedded>:11:580425)
    at /Applications/Atom Nightly.app/Contents/Resources/app/static/<embedded>:11:581678
    at ELECTRON_ASAR.js:530:9
    at FSReqWrap.wrapper [as oncomplete] (fs.js:658:17)
  at /app.asar/node_modules/language-rust-bundled/grammars/tree-sitter-rust.json:1:1
```

/fyi @smashwilson (since we discussed this issue in chat)

**Comment by jasonrudolph** [comment]
@maxbrunsfeld: Could this exception be due to 773209ece4e6063b53e776ce8a16891536ee879f?

**Comment by maxbrunsfeld** [comment]
I *think* this is fixed by https://github.com/atom/atom/commit/0e675d1c478c4869c75a8692f9b0631c956572d1, but I'll wait to close until someone else builds and sees it work, because for some reason it already worked on my machine.

**Comment by jasonrudolph** [comment]
> I _think_ this is fixed by [0e675d1](https://github.com/atom/atom/commit/0e675d1c478c4869c75a8692f9b0631c956572d1), but I'll wait to close until someone else builds and sees it work

Confirmed. Thanks, @maxbrunsfeld.
